savant/config/module_config.py

```diff
@@ -307,12 +347,21 @@ def configure_pipeline_elements(module_cfg: DictConfig) -> None:
         module_cfg.pipeline = OmegaConf.structured(Pipeline)
         return
 
+    logger.debug('Configure pipeline source...')
```

Comment:
```suggestion
    logger.debug('Configuring pipeline source...')
```

Same for https://github.com/insight-platform/Savant/pull/573/files#diff-0e0d7e987cb17ddd4aff5ffe31878d15fdc23cc58e6d4826894bab5c1708c5e4R355